coalib/misc/Constants.py

```diff
@@ -13,6 +13,11 @@
                  "happen. When asked for, the following information "
                  "may help investigating:")
 
+VERSION_CONFLICT_MESSAGE = ("There is a conflict in the version of a package "
+                            "you have installed and the requirements of "
+                            "coala. The following information may help "
+                            "to resolve this conflict: ")
```

Comment:
> I think the user can just google it.

Well you see how we struggle to find a proper solution if we had a recommendation that'd be great for the users.